thrust/system/cuda/vector.h

```diff
@@ -54,101 +51,7 @@ namespace cuda_cub
  *  \see device_vector
  */
 template<typename T, typename Allocator = allocator<T> >
-  class vector
-    : public thrust::detail::vector_base<T,Allocator>
-{
-  /*! \cond
-   */
-  private:
-    typedef thrust::detail::vector_base<T,Allocator> super_t;
-  /*! \endcond
-   */
-
-  public:
-
-  /*! \cond
-   */
-    typedef typename super_t::size_type  size_type;
-    typedef typename super_t::value_type value_type;
-  /*! \endcond
-   */
-
-    /*! This constructor creates an empty \p cuda_bulk::vector.
-     */
-    vector();
```

Comment:
These methods still have implementations in `thrust/system/cuda/detail/vector.inl` that need to removed.

Looks like that entire file can be deleted.